packages/config/tailwind-preset.js

```diff
@@ -82,6 +82,14 @@ module.exports = {
     require("@tailwindcss/typography"),
     require("tailwindcss-animate"),
   ],
+  safelist: [
+    {
+      pattern: /text-(blue|amber|red|green|indigo)-11/,
+    },
+    {
+      pattern: /border-(blue|amber|red|green|indigo)-8/,
+    },
+  ],
```

Comment:
I think it's better to describe all the classnames that will be used than maintaining the list here